backends/arm/test/common.py

```diff
@@ -137,7 +151,29 @@ def get_u55_compile_spec_unbuilt(
             "ethos-u55-128",
             system_config="Ethos_U55_High_End_Embedded",
             memory_mode="Shared_Sram",
-            extra_flags=None,
+            extra_flags="--debug-force-regor --output-format=raw",
```

Comment:
do we always want to keep `debug-force-regor` on?

Reply:
Yes, it's the way forward. We are enabling the new backend for U55 on the tests to sort out the last bits. The example still runs without the flag, so only affecting the unit tests at this point.